shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/value/identifier/IdentifierValue.java

```diff
@@ -27,13 +27,22 @@
  */
 @Getter
 public final class IdentifierValue implements ValueASTNode<String> {
-    
```

Comment:
keep the four whitespaces in all these classes

Reply:
OK